.github/workflows/main.yml

```
@@ -176,8 +182,9 @@ jobs:
       - name: Install the Python dependencies
         if: steps.filter.outputs.extension == 'true'
         run: |
+          python -m pip install --upgrade pip
           python -m pip install jupyter_packaging
-          python -m pip install jupyterlab
+          python -m pip install --pre jupyterlab
```

Do we want to keep using the `pre` release?

```suggestion
          python -m pip install jupyterlab
```

---

Good catch, thanks!